docs/install.md

```diff
@@ -752,41 +752,27 @@ The PMEM-CSI scheduler extender and admission webhook are provided by
 the PMEM-CSI controller. They need to be enabled during deployment via
 the `--schedulerListen=[<listen address>]:<port>` parameter. The
 listen address is optional and can be left out. The port is where a
-HTTPS server will run. It uses the same certificates as the internal
-gRPC service. When using the CA creation script described above, they
-will contain alternative names for the URLs described in this section
-(service names, `127.0.0.1` IP address).
-
-This parameter can be added to one of the existing deployment files
-with `kustomize`. All of the following examples assume that the
-current directory contains the `deploy` directory from the PMEM-CSI
-repository. It is also possible to reference the base via a
-[URL](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md).
-
-``` ShellSession
-$ mkdir my-pmem-csi-deployment
-
-$ cat >my-pmem-csi-deployment/kustomization.yaml <<EOF
-bases:
-  - ../deploy/kubernetes-1.16/lvm
-patchesJson6902:
-  - target:
-      group: apps
-      version: v1
-      kind: StatefulSet
-      name: pmem-csi-controller
-      namespace: pmem-csi
-    path: scheduler-patch.yaml
-EOF
-
-$ cat >my-pmem-csi-deployment/scheduler-patch.yaml <<EOF
-- op: add
-  path: /spec/template/spec/containers/0/command/-
-  value: "--schedulerListen=:8000"
-EOF
-
-$ kubectl create --kustomize my-pmem-csi-deployment
-```
+HTTPS server will run.
+
+The webhook needs TLS certificates which must be created in
+advance. The YAML files expects them in a secret called
+`pmem-csi-intel-com-controller-secret` and will not work without one.
+The operator is more flexible and creates a driver without the
+scheduler extensions by default. This can be changed by setting the
+`controllerTLSSecret` field in the `PmemCSIDeployment` API.
+
+That secret must contain the following data items:
+- `ca.crt`: root CA certificate
+- `tls.key`: secret key of the webhook
```

Comment:
Wouldn't be better to mention that the secret/public key is for both webhook and scheduler extender.

Reply:
The scheduler extender is also a webhook. Both run in the controller.

With that in mind, it's better to say above that "The *controller* needs TLS certificates" - i'll use that instead. The name of the API was already chosen accordingly.